packages/components/addon/components/hds/table/index.hbs

```diff
@@ -33,8 +34,9 @@
     {{else}}
       {{yield
         (hash
-          Tr=(component "hds/table/tr")
+          Tr=(component "hds/table/tr" isSelectable=@isSelectable)
           Th=(component "hds/table/th")
+          ThSelectable=(component "hds/table/th-selectable")
```

Comment:
Now that the `ThSelectable` is controlled directly via the `Tr` element, we don't need (and want) to yield it anymore.
```suggestion
```